posthog/session_recordings/session_recording_playlist_api.py

```diff
@@ -144,6 +144,7 @@ class Meta:
             "last_modified_at",
             "last_modified_by",
             "recordings_counts",
+            "type",
```

Comment:
ah, it's the addition here, causing the API tests to need updating 

totally valid test fails 👍